12/umbraco-cms/tutorials/creating-custom-views-for-blocklist.md

```diff
@@ -198,7 +198,7 @@ Now that we have updated our view and stylesheet let's assign it to our block.
 1. Go to **Product** in the **Settings** tree.
 2. Click the `cog` wheel next to **Features**.
 3. Select the **Product - Features - Block List**. The **Editor Settings** window opens.
-4. Select **Feature** from the **Availabe Blocks** configuration. The **Configuration of 'Feature'** window opens.
+4. Select **Feature** from the **Available Blocks** configuration. The **Configuration of 'Feature'** window opens.
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?